topics/event-classes.md

```diff
@@ -67,10 +67,12 @@ This page is broken up into sections for different areas of Talis Aspire.
 | `player.view` | A resource was opened in the player by a student. `digitisation.view` is also recorded by the Talis Elevate player when the content has originated from Copyright Clearance. This is in addition to `player.view`. |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
 | `player.view.contributors` | A resource was opened in the player by a contributor (academic, teaching assistant) |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view-contributors)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
 | `player.annotate.createannotation` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.annotate.createannotation.createcomment` | A new class annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.annotate.createannotation.personalnote` | A new personal annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

Comment:
A minor thing but personal notes are not threaded and you can't reply to existing class comment threads with them either.

```suggestion
| `player.annotate.createannotation.personalnote` | A new personal annotation (private note) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```